src/components/OverflowMenu/OverflowMenu-story.js

```diff
@@ -56,4 +56,28 @@ storiesOf('OverflowMenu', module)
         />
       </OverflowMenu>
     )
+  )
+  .addWithInfo(
+    'custom icons',
```

Comment:
Can we change this to say `custom trigger` since it's showing an example without an icon?